crates/hyperdrive-math/src/lib.rs

```diff
@@ -406,7 +423,7 @@ mod tests {
             );
             counter += 1;
         }
-        assert!(counter >= 1_000); // this passed at least 1,000 times
+        assert!(counter >= 5_000); // this passed at least 5,000 times
```

Comment:
We have some `continue` conditions in the test: when we adjust the random state to have zero exposure and when we adjust it again with the output of `reserves_given_rate_ignoring_exposure`. This counter assert is just making sure that the continues don't lead us to think the test is passing when it's actually skipping all the time. So worst case scenario we pass 5k times, best case scenario we pass 10k times.

Reply:
I thought about using the actual variable (`FAST_FUZZ_RUNS / 2`) but if you try to run that code you'll see there are some type issues. We need to do type conversions because `FAST_FUZZ_RUNS` is an int, and it ends up being v clunky. I'll adjust the comment and lmk what you think.

Reply:
Ah, yeah works for me!